collections_to_communities.json

```diff
@@ -0,0 +1,328 @@
+[
+  {
+    "title": "Biostatistics Collaboration Center Lecture Series",
```

Comment:
"title" and "actions" are remnants of a data model I had in mind that won't be used. They can be ignored.

Reply:
Well, I guess I should have checked this comment before replying on slack.